docs/usage.md

```diff
@@ -25,6 +25,11 @@ The following pointers help understand the various steps of this lifecycle:
 
 5. Content developer selects the CR again and clicks **Submit Changes for Review**.
 
+    <table>
+        <th>NOTE</th>
+        <td>It is strongly recommended not to rename the records <em>after</em> the changes have been submitted for review. However, if you have to rename, refer to the section <a href="#known-issues">Known Issues</a>.</td>
```

Comment:
We should not call this as a "known-issues". Can we call it as Recommended Step or Steps to push renamed file/folder.

Reply:
Changed